target/linux/ar71xx/files/arch/mips/ath79/mach-tl-wr840n-v2.c

```diff
@@ -0,0 +1,127 @@
+/*
+ *  TP-LINK TL-WR841N/ND v9/v11 / TL-WR842N/ND v3
```

Comment:
You're right, I'm sorry.
The linker takes only used variables/functions from the object file to the kernel.